scripts/register_oauth_scopes.py

```diff
@@ -1,24 +1,23 @@
 """
-Register the list of OAuth2 scopes that can be requested by third parties. This populates the Mongo collection
-    referenced by CAS when responding to authorization grant requests.
-
-The database class is minimal; the exact specification for what a scope contains lives in the
-    python module from which this collection is drawn.
+Register the list of OAuth2 scopes that can be requested by third parties. This populates the Postgres collection
+referenced by CAS when responding to authorization grant requests. The database class is minimal; the exact
+specification for what a scope contains lives in the python module from which this collection is drawn.
 """
+
 import sys
 import logging
 
-from modularodm import Q
-from modularodm.exceptions import NoResultsFound
+import django
+from django.db import transaction
+
+django.setup()
 
 from scripts import utils as script_utils
 
 from framework.auth import oauth_scopes
-from framework.mongo import set_up_storage, storage
-from framework.transactions.context import TokuTransaction
-
+from osf.models import ApiOAuth2Scope
```

Comment:
Unless you're checking `isinstance`,`issubclass`, or `==` then no. They should have the same behavior for querying.